scripts/fbi/hate_crime/preprocess_aggregations.py

```diff
@@ -0,0 +1,1796 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A script to process FBI Hate Crime data."""
```

Comment:
Please add a section to the README describing how to run this script and what output it generates.

Reply:
added details about running the script. I can't find the script I had used to rename stat vars, but it was a short script using mcf-edit tool.

Reply:
Can you please add a note to the readme indicating the statvars need to be modified after this script is run?